Wikipedia/Localizations/qqq.lproj/Localizable.strings

```diff
@@ -434,18 +434,15 @@
 "top-read-header-generic" = "Wikipedia\n{{Identical|Wikipedia}}";
 "reference-title" = "Title shown above reference/citation popover. $1 is replaced with the reference link text - i.e. '[1]'\n{{Identical|Reference}}";
 "in-the-news-notification-read-now-action-title" = "Title on the 'Read Now' action button";
-"in-the-news-notification-share-action-title" = "Title on the 'Share' action button";
+"in-the-news-notification-share-action-title" = "Title on the 'Share' action button\n{{Identical|Share}}";
 "in-the-news-notification-save-for-later-action-title" = "Title on the 'Save for later' action button";
 "in-the-news-currently-trending" = "Currently trending - indicates that the story is trending right now";
 "in-the-news-title" = "Title for the 'In the news' notification & feed section";
 "in-the-news-title-for-date" = "Title for news on a given date - $1 is replaced with the date";
-
 "number-thousands" = "$1K - $1 is replaced with a number represented in thousands. In English the letter 'K' is commonly used after a number to indicate that number is in 'thousands'. So the letter 'K' should be changed to a character or short string indicating thousands. For example 500,000 would become 500K. If there is no simple translation for this in the target language, make the translation $1 with no other characters and the full number will be shown.";
 "number-millions" = "$1M - $1 is replaced with a number represented in millions. In English 'M' is commonly used after a number to indicate that number is in 'millions'. So the 'M' should be changed to a character or short string indicating millions. For example 500,000,000 would become 500M. If there is no simple translation for this in the target language, make the translation $1 with no other characters and the full number will be shown.";
 "number-billions" = "$1B - $1 is replaced with a number represented in billions. In English 'B' is commonly used after a number to indicate that number is in 'billions'. So the 'B' should be changed to a character or short string indicating billions. For example 5,100,000,000 would become 5.1B. If there is no simple translation for this in the target language, make the translation $1 with no other characters and the full number will be shown.";
-
-"reference-title" = "Title shown above reference/citation popover. $1 is replaced with the reference link text - i.e. '[1]'\n{{Identical|Reference}}";
-
 "feed-news-notification-text" = "Text shown to users to notify them that it is now possible to get notifications for articles related to trending news";
 "feed-news-notification-button-text" = "Text for button to turn on trending news notifications";
-
+"explore-in-the-news-footer" = "MISSING DESCRIPTION; DO NOT COMMIT FILE YET";
```

Comment:
@montehurd @niedzielski my bad -they were unused, just forgot to remove them. Made the change.